File: app/src/main/java/com/nowfloats/manufacturing/API/model/AddTeams/ProfileImage.java
package com.nowfloats.manufacturing.API.model.AddTeams;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ProfileImage {

    @SerializedName("url")
    @Expose
    private String url;
    @SerializedName("description")
    @Expose
    private String description;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

}

File: inventoryorder/src/main/java/com/inventoryorder/model/doctorsData/Profileimage.kt
package com.inventoryorder.model.doctorsData

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Profileimage(
  @SerializedName("description")
  val description: String = "",
  @SerializedName("url")
  val url: String = ""
) : Serializable